queue/data.js

```diff
@@ -171,7 +171,7 @@ Task.prototype.hasClaim = function() {
  *
  * Returns a promise that all expired tasks have been deleted
  */
-Task.expire = async function(now) {
+Task.expire = async function(now = new Date()) {
```

Comment:
yeah, I just added it for fun... But it's probably safer to force people to specify `now`, as we always want some delay to be involved...